CommonCameraUtil/Components/PlayerMeshHandler.cs

```diff
@@ -40,6 +41,16 @@ public void Awake()
 		_animController = GetComponentInChildren<PlayerAnimController>();
 	}
 
+	public void Update()
+	{
+        IHatchlingOutfit hatchlingOutfit = CommonCameraUtil.Instance.HatchlingOutfit;
+		try
```

Comment:
Instead of a try catch can we just check if hatchlingOutfit is null or not

Reply:
yeah that's probably a better idea. i just pushed another commit